### DISCUSSION TITLE
Audio distorsion at the beginning of every song

### DISCUSSION THREAD

**Discussion by teoavv94** [mikebrady/shairport-sync#1696] [Q&A]
Hi,
I've an issue with my installation on Raspberry Pi 3 that I don't know how to resolve. The problem is that when the player (mainly Apple Music on iPhone 12 Pro Max, but it happens with every player and on every device) the audio is distorted when a new track starts. In my opinion it seems to be a buffer problem, so I've tried to increase the buffer size of alsa but the error persist. Thanks in advance for your help! ;)

For more context I paste the contents of my .config file, note that I'm using a bluetooth speaker with PulseAudio:

```
general =
{
	name = "JBL GO 3";
	interpolation = "auto";
	alac_decoder = "apple";
	audio_backend_latency_offset_in_seconds = -0.125;
	regtype = "_airplay._tcp";
};

sessioncontrol =
{
	allow_session_interruption = "yes";
};

alsa =
{
	output_device = "default";
	output_rate = 44100;
	output_format = "S16";
	disable_standby_mode = "auto";
	buffer_size = 2048;
};
```

**Comment by mikebrady** [comment]
Thanks. This is really an issue rather than a discussion item, IMHO. When you open an issue, you are asked for some information, which might have helped here.

Anyway, one extra thing that might be useful is to enable the logging of statistics and gather them for a while, especially through track-to-track transitions.

**Reply by teoavv94** [reply]
Thank you Mike! I've opened an issue. I close this discussion.
